src/test/java/uk/gov/justice/hmpps/casenotes/services/ExternalApiServiceTest.kt

```diff
@@ -268,13 +267,13 @@ class ExternalApiServiceTest {
         check {
           val components = UriComponentsBuilder.fromUriString(it).build()
           assertThat(components.path).isEqualTo("/api/offenders/{offenderIdentifier}/case-notes/v2")
-          assertThat(components.queryParams).containsExactlyInAnyOrderEntriesOf(
-            mapOf(
-              "size" to listOf("20"),
-              "page" to listOf("5"),
-              "sort" to listOf("creationDateTime,ASCsort=occurrenceDateTime,ASC"),
-            ),
-          )
+          assertThat(components.queryParams)
+            .containsEntry("size", listOf("20"))
+            .containsEntry("page", listOf("5"))
+            .containsKey("sort")
+            .hasSize(3)
+          assertThat(components.queryParams.get("sort"))
+            .containsExactly("creationDateTime,ASC", "occurrenceDateTime,ASC")
```

Comment:
note that we are concerned about ordering of the list here - otherwise they could be sorted in the wrong order